src/Microsoft.AspNetCore.Mvc.Razor.Host/GeneratedTagHelperAttributeContext.cs

```diff
@@ -18,5 +18,15 @@ public class GeneratedTagHelperAttributeContext
         /// Name the method to create <c>ModelExpression</c>s.
         /// </summary>
         public string CreateModelExpressionMethodName { get; set; }
+
+        /// <summary>
+        /// Instance name of the IModelExpressionProvider
```

Comment:
- Always end doc comment sentences with a period.
- Though this class is a lousy example, start every property description with either "Gets" or "Gets or sets".
- Use `<see cref="whatever"/>` when mentioning a C# interface, property, et cetera.
- What is an "Instance name"?

So this comment should be `Gets or sets the name of the <see cref="IModelExpressionProvider" />.`

Similar comments apply below and likely elsewhere in this PR.